tutorials/Dimension_Handling.ipynb

```diff
@@ -0,0 +1,4154 @@
+       "         1.45631540e+01],\n",
+       "       [ 1.75553287e+01,  1.96506539e+01,  4.20757053e+00,\n",
+       "         1.03412492e+00,  2.22919631e+00,  1.87921640e+01,\n",
+       "         2.03776121e+01,  3.77641474e+00,  6.30355657e+00,\n",
+       "         1.03712249e+01],\n",
+       "       [ 5.97467154e+00,  9.83056417e+00,  1.27625714e+01,\n",
+       "         9.66740698e+00,  1.33053421e+01,  1.22353186e+01,\n",
+       "        -9.89714911e-01,  4.94891473e+00,  1.18427111e+01,\n",
+       "         9.44907435e+00],\n",
+       "...\n",
+       "       [ 1.38757704e+01,  1.83863686e+01,  1.03139981e+01,\n",
+       "         1.00442693e+01,  1.28952417e+01,  1.22352772e+01,\n",
+       "         9.71756996e+00,  6.57696791e+00,  1.10432330e+01,\n",
+       "         2.03478908e+01],\n",
+       "       [ 1.94901078e+01,  7.72110555e+00,  1.70172641e+01,\n",
+       "         1.16445538e+01,  4.56336210e+00,  1.42529800e+00,\n",
+       "        -6.77478357e-01,  1.55661625e+00,  3.83132564e+00,\n",
+       "         1.72249589e+01],\n",
+       "       [ 5.71416646e+00,  1.63894575e+01,  9.23409025e+00,\n",
+       "         3.54024807e+00,  1.63192885e+01,  1.82041529e+01,\n",
+       "         1.93933734e+01,  1.31416482e+01,  1.43947255e+01,\n",
+       "         1.08362201e+01],\n",
+       "       [ 9.24551408e+00,  2.16468824e+01,  5.20689186e-01,\n",
+       "         1.94635556e+01,  5.16272310e+00,  1.26412064e+01,\n",
+       "         1.04677155e+01,  1.23261874e+01,  1.18393369e+01,\n",
+       "         4.79039382e+00],\n",
+       "       [ 4.16452314e+00,  2.43650695e+00,  4.29515334e-01,\n",
+       "         2.05556446e+01,  6.45836848e+00,  2.87991004e+00,\n",
+       "         6.15523884e+00,  7.99932744e+00,  1.24329978e+01,\n",
+       "         2.98415626e+00]])\n",
+       "Coordinates:\n",
+       "  * valid_time      (valid_time) datetime64[ns] 2024-01-01 ... 2024-01-31\n",
+       "  * station_number  (station_number) int64 1 2 3 4 5 6 7 8 9 10"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "obs_stations = xr.DataArray(\n",
+    "    data=np.random.uniform(0, 20, size=[len(valid_times), len(station_numbers)]),\n",
+    "    dims=[\"valid_time\", \"station_number\"],\n",
+    "    coords={\"valid_time\": valid_times, \"station_number\": station_numbers},\n",
+    ")\n",
+    "fcst_stations = obs_stations + np.random.normal(size=[len(valid_times), len(station_numbers)])\n",
+    "fcst_stations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If we want to calculate the MSE aggregated across time and space, there are a couple of different ways that we can call the MSE function that will give us the same result.\n",
+    "1. Not using any dim handling arg will aggregate data and take the mean across all dimensions\n",
```

Comment:
At the moment this list isn't rendering as an indented numbered list. If it isn't meant to, no worries. If it is meant to render as an indented numbered list, then adding a line space beforehand might make it render properly. 

(I am not sure what happens if you  add a line space and also leave the indentation before 1. in the code though - you might need to also remove the indentation as well)

Reply:
Thanks - I've added a space. Hopefully they render fine now